nova.sh

```diff
@@ -103,6 +103,51 @@ create_patch() {
     file patches/nova.patch | grep UTF-16 && (iconv --from-code UTF-16 --to-code UTF-8 patches/nova.patch > x && mv x patches/nova.patch)
 }
 
+tracing_nova_add() {
```

Comment:
it inserts a debug message at the beginning of every method. I was unable to get a useful stack trace form the debugger, so that was the only thing that helped me here. It's hacky but it always works, also on linux without any debugger, and its only temporary